packages/tailwind/src/tailwind.spec.tsx

```diff
@@ -422,3 +423,38 @@ describe("<Tailwind> component", () => {
     );
   });
 });
+
+describe("Tailwind component", () => {
```

Comment:
Love that you made a test for this, thank you so much/

Was there any reason you made a new `describe` for this instead of using the other one?

Reply:
Ah I was following what I thought was convention because there were two top level Tailwind tests. I've moved it up to be alongside the other tests that check for component rendering behaviour.